feature/communitylist/src/main/java/com/iteneum/communitylist/presentation/CommunityListView.kt

```diff
@@ -0,0 +1,41 @@
+package com.iteneum.communitylist.presentation
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.lazy.grid.items
+import androidx.compose.runtime.Composable
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.iteneum.designsystem.components.LPGenericElevatedCardImage
+import com.iteneum.designsystem.theme.LeasePertTheme
+import com.iteneum.designsystem.utils.TextUtils.TWO
+
+@Composable
+fun CommunityListView(
+    viewModel: CommunityListViewModel = hiltViewModel()
+) {
+    CommunityListContent(
+        state = viewModel.uiState
+    )
+}
+
+@Composable
+fun CommunityListContent(
+    state: CommunityListUIState
+) {
+    val dp8 = LeasePertTheme.sizes.smallerSize
+    LazyVerticalGrid(
+        columns = GridCells.Fixed(TWO),
+        verticalArrangement = Arrangement.spacedBy(dp8),
+        horizontalArrangement = Arrangement.spacedBy(dp8)
+    ) {
+        items(state.data ?: listOf()) { item ->
```

Comment:
It is a better way to handle the null safety if the list is null?

Reply:
We can add an empty state when we don't have information but this code will be changed